### PR TITLE
✨ Add new amp-ad type for verizonmedia

### PR DESCRIPTION
**Pull request by ebrandmark2**

### CODE REVIEW

File: ads/vendors/verizonmedia.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (alanorozco) [inline ads/vendors/verizonmedia.md:2]:
```suggestion
Copyright 2021 The AMP HTML Authors. All Rights Reserved.
```

Comment (ebrandmark2) [reply ads/vendors/verizonmedia.md]: updated and updated the verizonmedia.js copyright too
Thanks
Ed

File: examples/verizonmedia.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+<head>
+  <meta charset="utf-8">
+  <title>Verizon Ad Test</title>
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async custom-element="amp-ad" src="https://cdn.ampproject.org/v0/amp-ad-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+<h2>Verizon Ad Test No Fallback</h2>
+<amp-ad
```
Comment (calebcordry) [inline examples/verizonmedia.html:12]: Could you please move this example to `examples/amp-ad/ads.amp.html`?

File: ads/_config.js [hunk 3]
```diff
@@ -1233,6 +1233,11 @@ const adConfig = jsonConfiguration({
     renderStartImplemented: true,
   },
 
+  'verizonmedia': {
+    prefetch: 'https://jac.yahoosandbox.com/amp/jac.js',
+    preconnect: ['https://jill.fc.yahoo.com'],
```
Comment (calebcordry) [inline ads/_config.js:1238]: I see you are not using the renderStart API, have you looked into it? It should give you some performance gains but is optional.

**Review by calebcordry — APPROVED** [review]

**Review by calebcordry — CHANGES_REQUESTED** [review]
sorry accidental approve click. Thanks for contributing, just a couple of small questions.

**Comment by calebcordry** [comment]
also @powerivq could you please confirm this is the right new file structure? We might need to update the docs after your migration.

**Comment by powerivq** [comment]
@calebcordry yeah that is correct.

File: ads/_config.js (hunk 3 above)
Comment (ebrandmark2) [reply ads/_config.js:1238]: great thanks.  will add it and include in nexxt push

File: examples/verizonmedia.html (hunk 2 above)
Comment (ebrandmark2) [reply examples/verizonmedia.html:12]: done and re-pushing code now

**Review by calebcordry — APPROVED** [review]

**Comment by ebrandmark2** [comment]
So I see one approval and one pending.  Just wondering if you need anything else from me and the timeline of getting this approved and Merged.  

Thanks!
Ed Brandmark
VerizonMedia

**Comment by powerivq** [comment]
@ebrandmark2 build has failed unfortunately. can you rebase and push?

**Comment by ebrandmark2** [comment]
Thanks - not sure what to do here - git rebase master said everything was up to date.
Sorry for the lack of git knowledge - but wasn't sure what else to do - did do another git push origin - but keeps saying nothing changed.

**Comment by powerivq** [comment]
@ebrandmark2 
Almost right. You just need to do a `git pull origin master` first.

**Comment by alanorozco** [comment]
> @ebrandmark2 So I see one approval and one pending. Just wondering if you need anything else from me and the timeline of getting this approved and Merged.

To clarify, my pending review is not required for this change.

**Comment by powerivq** [comment]
@ebrandmark2 

You can run `gulp lint --fix` to fix the linter issues you now have.

File: ads/vendors/verizonmedia.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  writeScript,
+  validateData,
+} from '../../3p/3p';
+
+export function verizonmedia(global, data) {
```
Comment (kristoferbaxter) [inline ads/vendors/verizonmedia.js:22]: Can we correctly denote the types for these objects?

`global` is a `Window`.
`data` is `{{config: string}}`.

Additionally, this adds a new global for the `ads.externs.js` file `window.jacData`.

Comment (ebrandmark2) [reply ads/vendors/verizonmedia.js:22]: updated the objects.  not sure what you want for the window.jacData.  Pushed the update.
thanks

Comment (powerivq) [reply ads/vendors/verizonmedia.js:22]: @ebrandmark2 Please add a line in https://github.com/ampproject/amphtml/blob/master/ads/ads.extern.js with

```
// verizonmedia.js
window.jacData
```

It helps the type check.

**Comment by powerivq** [comment]
https://app.circleci.com/pipelines/github/ampproject/amphtml/3335/workflows/4313b20f-7ab9-4bcb-9551-45fa32d446e5/jobs/40311 Still needs to fix this prettifier error on the documentation.

Also, just happened to see the CLA check also failed. `ebrandma` also needs to sign a CLA since there are commits attributed to that account.

**Review by powerivq — APPROVED** [review]

**Pull request merged**